backend/src/main/java/com/bakdata/conquery/models/datasets/concepts/filters/postalcode/PostalCodesManager.java

```diff
@@ -0,0 +1,102 @@
+package com.bakdata.conquery.models.datasets.concepts.filters.postalcode;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.zip.GZIPInputStream;
+
+import com.github.powerlibraries.io.In;
+import com.univocity.parsers.csv.CsvParser;
+import com.univocity.parsers.csv.CsvParserSettings;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class PostalCodesManager {
+
+	@Getter
+	@Setter
+	private String dataFilePath = null;
+
+	@Getter
+	private List<Double> data;
+
+	private int recordsNumber = 0;
+
+	public PostalCodesManager() {
+		try {
+			loadData();
+		}
+		catch (IOException e) {
+			e.printStackTrace();
+		}
+		catch (URISyntaxException e) {
+			e.printStackTrace();
+		}
```

Comment:
Exceptions einfach zu schlucken ist nicht gut. Wenn was schiefgeht wissen wir das ggf gar nicht. Und das hier ist ein schwerwiegender Fehler: Das Objekt kann nicht initialisiert werden.

Reply:
okay dann tue ich es nachdem ich den kompletten fertig habe, da die aktuelle gezippte Datei einige fehlerhaften Zeilen hat

Reply:
Du kannst einfach die Exceptions werfen lassen. Das ist vollkommen in Ordnung. Der Code, der die Klasse lädt sollte das exception handling machen. An manchen Stellen sind Exceptions ja zB in Ordnung, an anderen wiederum nicht, das müssen die selber entscheiden. Sie zu schlucken ist keine gute idee.